flowey/flowey_lib_common/src/install_dist_pkg.rs

```diff
@@ -131,16 +217,9 @@ impl FlowNode for Node {
                         return Ok(false)
                     }
 
-                    let sh = xshell::Shell::new()?;
-
                     let mut installed_packages = BTreeSet::new();
                     let packages_to_check = &packages;
-
-                    let fmt = "${binary:Package}\n";
-                    let output = xshell::cmd!(sh, "dpkg-query -W -f={fmt} {packages_to_check...}")
-                        .ignore_status()
-                        .output()?;
-                    let output = String::from_utf8(output.stdout)?;
+                    let output = pacman.query_cmd(packages_to_check)?;
                     for ln in output.trim().lines() {
```

Comment:
as discussed on the previous PR: query_cmd should be responsible for parsing the stdout of the particular package manager being used

Reply:
Done